pkg/steps/output_image_tag.go

```diff
@@ -72,7 +72,7 @@ func (s *outputImageTagStep) run(ctx context.Context, dry bool) error {
 }
 
 func (s *outputImageTagStep) Requires() []api.StepLink {
-	return []api.StepLink{api.InternalImageLink(s.config.From), api.ReleaseImagesLink()}
```

Comment:
This step promotes a `pipeline` ImageStreamTag into something else. I have no idea why this *ever* relied on the `stable` stream existing. The original author isn't of any help in understanding this.